controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java

```diff
@@ -1901,12 +1902,17 @@ private HttpResponse submit(String tenant, String application, HttpRequest reque
                                                             sourceRevision,
                                                             authorEmail,
                                                             sourceUrl,
-                                                            commit,
                                                             projectId,
                                                             applicationPackage,
                                                             dataParts.get(EnvironmentResource.APPLICATION_TEST_ZIP));
     }
 
+    private HttpResponse removeProdAllDeployments(String tenant, String application) {
```

Comment:
Consider naming it `removeAllProdDeployments`.

Reply:
Done